projects/dot-custom-elements/src/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html dir='ltr' lang='en'>
+    <head>
+        <meta charset='utf-8' />
+        <meta
+            name='viewport'
+            content='width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0'
+        />
+        <title>Stencil Component Starter</title>
+        <script src='/build/mycomponent.js'></script>
+        <script>
+            window.onload = function() {
```

Comment:
Do we need to wait for the load?

Reply:
yes, otherwise move the script to the body

Reply:
this is positioned in the `header` if we remove the `onLoad`, then I'll have to move the `script` tag to the *Body* after the declaration of the custom tags

Reply:
Oh I see... well for performance, all JS should go before `</boby>`